api/src/pcapi/routes/native/v1/account.py

```diff
@@ -69,6 +70,12 @@ def patch_user_profile(
 ) -> serializers.UserProfileResponse:
     profile_update_dict = body.dict(exclude_unset=True)
 
+    if (
+        "postal_code" in profile_update_dict
+        and profile_update_dict["postal_code"] in postal_code_utils.INELIGIBLE_POSTAL_CODES
+    ):
```

Comment:
Détail : proposition plus simple : 
```suggestion
    if profile_update_dict.get("postal_code") in postal_code_utils.INELIGIBLE_POSTAL_CODES:
```